server/seeder.js

```diff
@@ -0,0 +1,71 @@
+const faker = require('faker');
+const {db, Reviews} = require('./db.js');
+
+// drop database
+db.collections['reviews'].drop(() => {
+  console.log('reviews db dropped');
+
+  var formatMonth = function (month) {
+    if (month === '01') {
```

Comment:
+1 on not storing dates as Strings, but even if you did, this could be cleaned up a lot by using an object.
```
var monthAbbreviation = {
  01: 'Jan',
  02: 'Feb',
  ...
};
return monthAbbreviation[month];
```

Reply:
An obj! Good point so much easier! I went back and stored as a date, I'll handle the formatting on the client side.